tracing/logger_test.go

```diff
@@ -0,0 +1,73 @@
+package tracing
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"go.dedis.ch/kyber/v3/suites"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/honeycombio/beeline-go"
+
+	"go.dedis.ch/onet/v3/log"
+
+	"go.dedis.ch/onet/v3"
+)
+
+var tSuite = suites.MustFind("Ed25519")
+
+func TestLoggingRealHC(t *testing.T) {
+	t.Skip("If you want to run this, get an API key from honeycomb.io")
+	os.Setenv("HONEYCOMB_API_KEY", "API_KEY:Dataset")
+	log.AddUserUninterestingGoroutine("runtime_pollWait")
+	log.AddUserUninterestingGoroutine("writeLoop")
+	log.AddUserUninterestingGoroutine("worker")
+	local := onet.NewLocalTest(tSuite)
+	server := local.GenServers(1)[0]
+	if server.UnauthOk {
+		log.Print("OK to unauth")
+	}
+	time.Sleep(time.Second)
+	defer local.CloseAll()
+	defer beeline.Close()
```

Comment:
`defer` at the end?

Reply:
oups - that was supposed to go on afterwards... But then I wrote the `simulCollector` and didn't needed it.